cli/cmdlineparser.cpp

```diff
@@ -1084,6 +1093,9 @@ void CmdLineParser::printHelp()
         "                         be considered for evaluation.\n"
         "    --config-excludes-file=<file>\n"
         "                         A file that contains a list of config-excludes\n"
+        "    --disable=<id>       Disable individual checks.\n"
```

Comment:
hmm.. what do you think.. would it be better to write "severities" instead of "checks" perhaps. Also in the `--enable` description?

Reply:
This mixes checks and severities to begin with which might be confusing. But I don't think there's need to adjust this right now.